src/modules/index.js

```diff
@@ -31,8 +51,9 @@ const popupPlaceClose = document.querySelector(".popup__close");
 const popupUserInfo = document.querySelector(".popup__user-info");
 const userAvatar = document.querySelector(".user-info__photo");
 const avatarButton = document.querySelector(".popup__button_save-avatar");
+const serverUrl = NODE_ENV ==="development" ? "http://praktikum.tk/cohort7" : "https://praktikum.tk/cohort7";
```

Comment:
Нужно исправить: при запуске сайта с хот релоудом используется подключение по протоколу `https` к API.
При запуске не назначается `NODE_ENV`:
https://github.com/MKvaratshelia/praktikum_mesto/blob/61fb4f9d247d5d89128d0d11f169bd5aaac5b480/package.json#L8

Так как во всех остальных командах передается только `NODE_ENV=production`:
https://github.com/MKvaratshelia/praktikum_mesto/blob/61fb4f9d247d5d89128d0d11f169bd5aaac5b480/package.json#L7-L9
то лучше изменить условие, чтобы осуществлялась проверка на `production` режим, а для разработки использовалась ветка _else_:
```suggestion
const serverUrl = NODE_ENV === "production" ? "https://praktikum.tk/cohort7" : "http://praktikum.tk/cohort7";
```